provisioning/miniprov/hare_mp/main.py

```diff
@@ -513,6 +566,7 @@ def init(args):
         # Stopping hax and consul
         hax_starter.stop()
```

Comment:
Check if hax and consul stop are getting logged, you can add logs in there respective functions which are present in 'consul_starter.py' and 'hax_starter.py'